src/class-wpml-elementor-urls.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+class WPML_Elementor_URLs implements IWPML_Action {
+
```

Comment:
@brucepearson, I thought you would also add the private properties `$element_factory` and `$domains`?